numpy/typing/tests/data/reveal/strings.pyi

```diff
@@ -42,9 +42,6 @@ assert_type(np.strings.decode(AR_S), npt.NDArray[np.str_])
 assert_type(np.strings.expandtabs(AR_U), npt.NDArray[np.str_])
 assert_type(np.strings.expandtabs(AR_S, tabsize=4), npt.NDArray[np.bytes_])
 
-assert_type(np.strings.join(AR_U, "_"), npt.NDArray[np.str_])
-assert_type(np.strings.join(AR_S, [b"_", b""]), npt.NDArray[np.bytes_])
-
```

Comment:
I'm curious to why you removed these tests.

Reply:
These functions don't exist at runtime 🤷🏻